s3_storage_provider.py

```diff
@@ -26,9 +26,15 @@
 from twisted.python.failure import Failure
 from twisted.python.threadpool import ThreadPool
 
+from synapse.logging.context import LoggingContext, make_deferred_yieldable
 from synapse.rest.media.v1._base import Responder
 from synapse.rest.media.v1.storage_provider import StorageProvider
-from synapse.util.logcontext import LoggingContext, make_deferred_yieldable
```

Comment:
This seems to be a deprecated import path last touched in July 2019 (1.2.0rc1).

Reply:
Although I'm now realizing I didn't even end up touching this line at the end of this PR. I can revert if we want.

Reply:
Seeing as this PR is about updating import paths I think it's fine to leave in :)